app/assets/javascripts/testing.js.erb

```diff
@@ -0,0 +1,3 @@
+<% if Rails.env == "test" %>
```

Comment:
I believe the `require_tree .` in application.js does it. Since it's loaded in all pages and we need to do it in all pages, it seems like the correct place.